modules/standard/HDFS.chpl

```diff
@@ -138,29 +443,36 @@ proc hdfsChapelFile.hdfsClose() {
 }
 
 // ------------- General Utils ----------
-// Returns the file for the locale that we are on when we query
+/* Returns the file for the locale that we are on when we query */
+pragma "no doc"
 proc hdfsChapelFile.getLocal(): file {
   return rcLocal(this.files);
 }
 
-// Convenience funtion. Does the same as file.reader except that we dont have to get
-// our local file first
+/* Convenience funtion. Does the same as file.reader except that we dont have
+   to get our local file first */
+pragma "no doc"
 proc hdfsChapelFile.hdfsReader(param kind=iokind.dynamic, param locking=true, start:int(64) = 0, end:int(64) = max(int(64)), hints:iohints = IOHINT_NONE) {
   return rcLocal(this.files).reader(kind, locking, start, end, hints);
 }
 
 // ------------- End mulitlocale ---------------
 
+pragma "no doc"
 record hdfsChapelFile_local {
   var home: locale = here;
   var _internal_:qio_locale_map_ptr_t = QIO_LOCALE_MAP_PTR_T_NULL;
 }
 
+pragma "no doc"
 record hdfsChapelFileSystem_local {
   var home: locale;
   var _internal_: c_void_ptr;
 }
 
+pragma "no doc"
+// MPPF - I'm not sure why this is here - it doesn't seem to be used
+// in the examples
```

Comment:
I'm guessing it overrides the other open definitions, so it can be used in hdfsChapelFileSystem_local.hdfs_chapel_open() below.